### PR TITLE
fetch extended local forecast

### PR DESCRIPTION
**Pull request by bretthev**
Gets extended local forecast and puts it in the store.

### CODE REVIEW

**Review by sikemausa — COMMENTED** [review]
Brett, your growth as a coder and person never ceases to amaze me. Fantastic work!

File: src/components/Header.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+const Header = ({ temp, main, name }) => {
   return(
   <div>
     <h2 className="page-title">Weather Tracker</h2>
-    <section id="header" onClick={() => onClick ()}>
-      <p id="current-local-forecast">Current forecast for <span id="city-name">{name}</span>: {temp} degrees and {description}</p>
-      <p id="view-extended-forecast">View extended forecast</p>
+    <section id="header">
+    <p id="current-local-forecast">{temp ? `Current forecast for ${name} : ${temp} degrees and ${main}` : 'Loading weather for your location...'}</p>
+      <Link to="/ExtendedForecast" id="view-extended-forecast">View extended forecast</Link>
```
Comment (kylem038) [inline src/components/Header.js:10]: Thanks for making this a link!

**Comment by sikemausa** [comment]
Butt

File: src/reducers/CurrentCityReducer.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    outlook: 'normal',
+  }
+}
+
+
+const currentCityReducer = ( state = initialState, action ) => {
+  switch(action.type) {
+    case "CURRENT_LOCAL_CITY":
+      return Object.assign({}, state, action.weather);
+    case "EXTENDED_LOCAL_CITY":
```
Comment (kylem038) [inline src/reducers/CurrentCityReducer.js]: Do we need this if there's an Extended reducer?

**Pull request merged**